.eslintrc.js

```diff
@@ -8,7 +8,61 @@ module.exports = {
     "strict": "off",
     "prefer-rest-params": "off",
     "react/require-extension" : "off",
-    "import/no-extraneous-dependencies" : "off"
+    "import/no-extraneous-dependencies" : "off",
+    "prefer-destructuring": "off",
+    "comma-dangle": "off",
+    "no-buffer-constructor": "off",
+    "no-var": "off",
+    "prefer-template": "off",
+    "prefer-spread": "off",
+
+    "arrow-parens": "off",
```

Comment:
Probably at some point these new rules should be reviewed one by one and potentially fixed/restored, but it's going to be much easier after this PR.